system-addon/content-src/styles/_theme.scss

```diff
@@ -16,10 +17,13 @@ body {
   --newtab-text-conditional-color: $grey-60;
   --newtab-text-primary-color: $grey-90;
   --newtab-text-secondary-color: $grey-50;
+  --newtab-textbox-background-color: $white;
   --newtab-textbox-border: $grey-90-20;
-  --newtab-textbox-color: inherit;
+  --newtab-textbox-focus-color: $blue-60;
+  --newtab-textbox-focus-boxshadow: 0 0 0 1px $blue-60, 0 0 0 4px rgba($blue-60, 0.3); // sass-lint:disable-line no-color-literals
```

Comment:
I don't like this but... is there a cleaner solution?

Reply:
Are you referring to the duplicate stuff between default and dark? Maybe something like:
```diff
diff --git a/system-addon/content-src/styles/_theme.scss b/system-addon/content-src/styles/_theme.scss
index eb04f699..97bccdcc 100644
--- a/system-addon/content-src/styles/_theme.scss
+++ b/system-addon/content-src/styles/_theme.scss
@@ -1 +1,6 @@
+@mixin textbox-focus($color) {
+  --newtab-textbox-focus-color: $color;
+  --newtab-textbox-focus-boxshadow: 0 0 0 1px $color, 0 0 0 4px rgba($color, 0.3); // sass-lint:disable-line no-color-literals
+}
+
 // Default theme
@@ -21,4 +26,3 @@ body {
   --newtab-textbox-border: $grey-90-20;
-  --newtab-textbox-focus-color: $blue-60;
-  --newtab-textbox-focus-boxshadow: 0 0 0 1px $blue-60, 0 0 0 4px rgba($blue-60, 0.3); // sass-lint:disable-line no-color-literals
+  @include textbox-focus($blue-60);
 
@@ -74,4 +78,3 @@ body {
   --newtab-textbox-border: $grey-10-20;
-  --newtab-textbox-focus-color: $blue-40;
-  --newtab-textbox-focus-boxshadow: 0 0 0 1px $blue-40, 0 0 0 4px rgba($blue-40, 0.3); // sass-lint:disable-line no-color-literals
+  @include textbox-focus($blue-40);
 
```